src/domain/QuestionList/QuestionCart.tsx

```diff
@@ -0,0 +1,55 @@
+import { DeleteOutlined } from '@ant-design/icons';
+import { Divider, Drawer, Space } from 'antd';
+import React from 'react';
+
+import { InterviewQuestionModel } from '../../services/Client';
+
+export default function QuestionCart({
+  isVisible,
+  visibilityChangedCallback,
+  addList,
+  removeList,
+  removeFromAddListCallback,
+  removeFromRemoveListCallback,
+}: {
+  isVisible: boolean;
```

Comment:
I recommend move definition of props into separate file. It's hard to read code now.